packages/vsce/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the "cics-extension-for-zowe" extension will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Inspect CICS Resource command is added to give CEMT-like behaviour for viewing resource information [#281](https://github.com/zowe/cics-for-zowe-client/issues/281)
```

Comment:
I guess it's hard to avoid CEMT but something like "New command to quickly inspect CICS resource information" might be nicer than making comparisons to technology new developers may not be as familiar with?

Reply:
Happy to do that!